MapboxNavigation/ErrorCode.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+/**
+ Enum used for indicating the type of error occured while speaking an instruction.
+ */
+public enum SpokenInstructionErrorCode: Int {
+    /**
+     Default error.
+     */
+    case defaultError = -1
+    
+    /**
+     The audio player failed to play audio data.
+     */
+    case spokenInstructionAudioPlayerFailedToPlay
```

Comment:
`spokenInstruction` is redundant here, because it’s part of an enumeration that contains `SpokenInstruction` in its name. Rename this case to `audioPlayerFailedToPlay`.